tools/tests/actions.js

```diff
@@ -50,7 +50,7 @@ function createAsyncSpec(callback = () => {}, frames = DEFAULT_FRAMES_TO_WAIT) {
 
 
 async function waitServerResponse(responseTimeout = 50) {
-	return new Promise((resolve, reject) => {
+	return new Promise((resolve) => {
 		setTimeout(resolve, responseTimeout);
 	})
 }
```

Comment:
If you want we can also rework now the `waitServerResponse` method to use the new xhr listeners. Like this will have a real wait for the server response instead of setting timeout :)
```
async function submitAndWaitForServerResponse(xhr, submitButton) {
	return new Promise((resolve, reject) => {
                xhr.addEventListener('loadend', resolve);
                xhr.addEventListener('error', reject);
                click(submitButton);
	})
}
```

Reply:
This is a great idea. I no longer use this function for the XHR test. It is used in the forms-control main tests. I want to avoid making such essential change to the other tests during this task, but I added a small task to change this as it is important - https://coherent-labs.atlassian.net/browse/COH-15687